core-bundle/src/Routing/FrontendLoader.php

```diff
@@ -61,7 +67,7 @@ public function supports($resource, $type = null): bool
      */
     private function addFrontendRoute(RouteCollection $routes, array $defaults): void
     {
-        $route = new Route('/{alias}%contao.url_suffix%', $defaults, ['alias' => '.+']);
+        $route = new Route('/{alias}'.$this->urlSuffix, $defaults, ['alias' => '.+']);
```

Comment:
What was the problem with the previous implementation?

Reply:
Because we're no longer using the route loader but manually use the class (to keep BC) the container parameters are not resolved. see https://github.com/contao/contao/pull/95/files#diff-5f8c741675801f711d0a61e756575413R51